stumpy/fastmath.py

```diff
@@ -0,0 +1,51 @@
+import importlib
+
+from stumpy import config
+
+
+def _set(module_name, func_name, flag):
```

Comment:
@NimaSarajpoor 
Add test function for the functions in this new module. 


Note: Ideally, the tests should have been written first!